connection.js

```diff
@@ -571,653 +492,172 @@ class Connection {
-            }
-                
-            // min
-            if ((config[minFamilyArray[0]] || config[minFamilyArray[1]] || config[minFamilyArray[2]]) && (speedString === blitzLiveCorrConverted)) {
-                minimumMaximumSentence = "Minimum ";
-                speedSentence = "for " + speedString + " ";
-                increaseReduceSentence = ", please increase ";
-                connBelowAboveSentence = " below ";
-                connSentence = user.username + " wanted " + minimumMaximumSentence; // example : "user wanted minimum "
-                if (config[minFamilyArray[0]] && !config[minFamilyArray[1]] && !config[minFamilyArray[2]]) {
-                    timeNumber = config[minFamilyArray[0]];
-                    timeToString = timespanToDisplayString(config[minFamilyArray[0]]);
-                    rankedUnrankedGamesIsSentence = "games is ";
-                }
-                if (config[minFamilyArray[1]] && notification.ranked) {
-                    timeNumber = config[minFamilyArray[1]];
-                    timeToString = timespanToDisplayString(config[minFamilyArray[1]]);
-                    rankedUnrankedGamesIsSentence = "ranked games is ";
-                }
-                if (config[minFamilyArray[2]] && !notification.ranked) {
-                    timeNumber = config[minFamilyArray[2]];
-                    timeToString = timespanToDisplayString(config[minFamilyArray[2]]);
-                    rankedUnrankedGamesIsSentence = "unranked games is ";
-                }
+        // t.max_time (fischer) to t.min_time ! (it doesnt exist !)
+        // 5) t.time_control possible values:
+        // "fischer" , "byoyomi" , "canadian", "simple", "absolute"
+        // 6) t.speed possible values: "blitz", "live", "correspondence"
+        // 7) the t.speed === "blitz"/"live"/"correspondence") test is for efficiency :
+        // if false, we skip the unrelevant arg and skip to next speed to test :
+        // ex: if ((arg family is minmaintimeblitz) AND (notification is live/correspondence)),
+        ////////////////////////////////////////////////////////////////////////////////////////
+
+        function UHMAEAT(mainPeriodTime) {
+            for (let blitzLiveCorr of ["blitz", "live", "corr"]) {
+                if (t.speed === convertBlitzLiveCorr(blitzLiveCorr)) {
+                    const minFamilyArray = familyArrayMIBLIsminIsmaxFromGeneralArgString("min" + mainPeriodTime + blitzLiveCorr);
+                    const maxFamilyArray = familyArrayMIBLIsminIsmaxFromGeneralArgString("max" + mainPeriodTime + blitzLiveCorr);
+                    // example : ["min(main/period)timeblitz", "min(main/period)timeblitzranked", "min(main/period)timeblitzunranked", 
+                    //            ["minimum", "increase", "below", "low"], [true, false]]
+
+                    minmaxRejectTests(minFamilyArray);
+                    minmaxRejectTests(maxFamilyArray);
```

Comment:
Shouldnt the values retuned from these functions be returned somehow? Possibly only if they are not `null`?

Small Js trick that could work is `return minmaxRejectTests(minFamilyArray) || minmaxRejectTests(maxFamilyArray);`